src/renderer/bootstrap.tsx

```diff
@@ -37,8 +37,8 @@ import { ExtensionDiscovery } from "../extensions/extension-discovery";
 import { ExtensionLoader } from "../extensions/extension-loader";
 import { ExtensionsStore } from "../extensions/extensions-store";
 import { FilesystemProvisionerStore } from "../main/extension-filesystem";
-import { App } from "./components/app";
-import { LensApp } from "./lens-app";
+import { ClusterFrame } from "./components/cluster-frame";
+import { RootFrame } from "./root-frame";
```

Comment:
How about `AppFrame|MainFrame` and `ClusterFrame`.

Reply:
I like `MainFrame`